### DISCUSSION TITLE
Steal slippers syntax for referring to component slots and component name

### DISCUSSION THREAD

**Discussion by gone** [django-components/django-components#122] [Ideas]
A sample component from the docs is:

```
{% component_block "calendar" date="2020-06-06" %}
    {% slot "body" %}{ slot.super }. Have a great day!{% endslot %}
{% endcomponent_block %}

```
In slippers syntax this would be:

```
{% #calendar date="2020-06-06"%}
  {% #body %}{ slot.super }. Have a great day!{% /body %}
{% /calendar %}
```
Slippers doesn't support the concept of extension, so slot.super wouldn't make sense - but it's nice to be able to refer to the section name directly

**Comment by EmilStenstrom** [comment]
I think this would be a too big of a change to Djangos template language. We have an long term goal to one day be included in Django itself, and this would contradict that goal I think.

**Comment by JuroOravec** [comment]
Closing as this has implemented in https://github.com/EmilStenstrom/django-components/releases/tag/0.90
